configs/nifi.properties

```diff
@@ -144,6 +146,18 @@ nifi.sensitive.props.algorithm=PBEWITHMD5AND256BITAES-CBC-OPENSSL
 nifi.sensitive.props.provider=BC
 nifi.sensitive.props.additional.keys=
 
+    {{if .Values.auth.ldap.enabled}}
+nifi.security.keystore=/opt/nifi/nifi-current/conf/{{.Values.properties.release}}-nifi-0.{{.Values.properties.release}}-nifi-headless.{{.Values.properties.namespace}}.svc.cluster.local/keystore.jks
```

Comment:
by passing `nifi-0` it will not work for a cluster of nifi. you can retrieve this value with hostname -f, and the variable should be injected in the propreties file.